.github/actions/constellation_create/action.yml

```diff
@@ -173,14 +173,15 @@ runs:
           --info logcollect.github.run-attempt="${{ github.run_attempt }}" \
           --info logcollect.github.ref-name="${{ github.ref_name }}" \
           --info logcollect.github.sha="${{ github.sha }}" \
-          --info logcollect.github.runner-os="${{ runner.os }}"
+          --info logcollect.github.runner-os="${{ runner.os }}" \
+          --force
```

Comment:
Is `--force` here just as a precaution or are we actually doing invalid configs in our tests?

Reply:
Currently the CLI always has the version of the last release. Right now `2.5.0`. (Should be 2.5.1?) However, the debug images are all on v2.6.0-pre. Our compatibility constraints say that the CLI is not compatible with newer images. Thus the validation fails. With bazel we will hopefully be able to give the CLI a valid pseudoversion, move it to a prerelease version as well and the `--force` flag can be removed from the action.

Let me put some version on this as a comment into the action.